### DISCUSSION TITLE
First install - not all plugins are available

### DISCUSSION THREAD

**Discussion by renocz** [SteamDeckHomebrew/decky-loader#287] [Q&A] [answered]
Hi guys, i started on steam deck a week ago and i love it. Found your app and wanted to test some plugins but a lot of them that are currently quoted like quicklaunch are not available in the stable build and the prerelease one. Any idea? 

Thanks a lot

**Comment by PartyWumpus** [comment, marked as answer]
A while ago the loader had a rewrite which made a lot of plugins no longer functional. Most of them were updated, but a few were not and so no longer work. At this point I'm pretty sure most of those have been replaced by similar tools.

For example, all (or most?) of Quicklaunch's features are available in Bash Shortcuts.

**Reply by renocz** [reply]
thanks a lot !
